website/docs/reference/project-configs/require-dbt-version.md

```diff
@@ -0,0 +1,91 @@
+---
+datatype: version-range
+default_value: None
+---
+## Definition
+
+Optionally specify a range of dbt versions that a project is compatible with.
+
+By using this configuration, dbt will throw a helpful error message if a user tries to run your project with an unsupported version of dbt. This is especially useful for packages (like [dbt-utils](https://github.com/fishtown-analytics/dbt-utils)). Additionally, this can help ensure that your whole team is synchronized on the same version of dbt for local development.
+
+If this configuration is not specified, no version check will occur.
+
+<Callout type="info" title="YAML Quoting">
+
+This configuration needs to be interpolated by the yaml parser as a string. As such, you should quote the value of the configuration. For example:
+* ✅`require-dbt-version: ">=0.16.0"`
+* ✅`require-dbt-version: '>=0.16.0'`
+* ❌ `require-dbt-version: >=0.16.0`
+
+</Callout>
+
+
+## Examples
+
+### Require a specific dbt version
+Use an exact version number. In this example, this project will only run with dbt v0.16.0 supported.
+
+<File name='dbt_project.yml'>
+
+```yml
+require-dbt-version: 0.16.0
+
+```
+
+</File>
+
+### Specify a minimum dbt version
+Use a `>=` operator for a minimum boundary. In this example, this project will run with any version of dbt greater than or equal to 0.16.0.
+
+
+<File name='dbt_project.yml'>
+
+```yml
+require-dbt-version: ">=0.16.0"
+
+```
+
+</File>
+
+
+### Pin to a minor dbt version using a range
+Use a comma separated list for an upper and lower bound. In this example, this project will run with dbt 0.16.x.
+
+<File name='dbt_project.yml'>
+
+```yml
+require-dbt-version: ">=0.16.0,<0.17.0"
```

Comment:
Does this need an array? Or are both valid? If both are valid, we should show both

Reply:
```suggestion
require-dbt-version: [">=0.16.0" ,"<0.17.0"]
```